webtau-test-server/src/main/java/org/testingisdocumenting/webtau/http/testserver/GraphQLResponseHandler.java

```diff
@@ -76,11 +111,19 @@ public void handle(String url, Request baseRequest, HttpServletRequest request,
         } else {
             response.setStatus(405);
         }
+    }
 
-        baseRequest.setHandled(true);
+    private boolean isAuthorised(HttpServletRequest request) {
```

Comment:
I know header is called "authorization", but I think what is happening here is Authentication?

Reply:
Yes, it's weird and confusing but it's not my invention: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization